apps/web/components/apps/installation/AccountsStepCard.tsx

```diff
@@ -1,17 +1,29 @@
+import type { TTeams } from "@pages/apps/installation/[[...step]]";
 import type { FC } from "react";
 import React, { useState } from "react";
 
 import { classNames } from "@calcom/lib";
 import { getPlaceholderAvatar } from "@calcom/lib/defaultAvatarImage";
 import { useLocale } from "@calcom/lib/hooks/useLocale";
-import type { Team, User } from "@calcom/prisma/client";
-import { Avatar, StepCard } from "@calcom/ui";
+import type { User } from "@calcom/prisma/client";
+import { StepCard } from "@calcom/ui";
+import { Avatar } from "@calcom/ui";
+
+export type PersonalAccountProps = Pick<User, "id" | "avatarUrl" | "name"> & { alreadyInstalled: boolean };
+
+type AccountStepCardProps = {
```

Comment:
NIT: Don't use colors like text-gray and use something like text-subtle which would work on both light mode and dark mode and add border radius

![Screenshot 2024-07-17 at 2 45 34 PM](https://github.com/user-attachments/assets/5be155fa-6d10-4469-bf16-90f510eebb28)